src/include/coredumpctl.h

```diff
@@ -1,3 +1,6 @@
+#ifndef __CEPH_COREDUMPCTL_H
```

Comment:
@tchaikov 
I just followed the std way all other files were.
But that is fine with me too.
I just added it because I was just there anyways.